app/components/tutorial.cjsx

```diff
@@ -42,15 +42,18 @@ module.exports = React.createClass
       TutorialComponent = this
 
       if tutorial.steps.length isnt 0
-        awaitTutorialMedia = tutorial.get 'attached_images'
-          .catch ->
-            # Checking for attached images throws if there are none.
-            []
-          .then (mediaResources) ->
-            mediaByID = {}
-            for mediaResource in mediaResources
-              mediaByID[mediaResource.id] = mediaResource
-            mediaByID
+        if tutorial.links.attached_images.ids?.length
```

Comment:
I didn't realize this until just now, but apparently whether a tutorial never has or had all the attached images deleted, if there are no attached images it looks like:
```
links
    attached_images
        href: "/tutorials/106/attached_images"
        type: "attached_images"
```
notably there's no `id` attribute at all, let alone the empty array I was expecting at minimum if the media was deleted.

so if I add the `isnt 0` I think the conditional gets met because `undefined` isnt `0` and it makes the request I don't want it to, but I'm not sure I'm interpreting things correctly.

maybe just `if tutorial.links.attached_images.ids?` would be preferred?